db/migrate/20160708235959_add_web_hook_event.rb

```diff
@@ -0,0 +1,18 @@
+class AddWebHookEvent < ActiveRecord::Migration
+  def change
+    enable_extension 'uuid-ossp' unless extension_enabled?('uuid-ossp')
+
+    create_table :web_hook_events, id: :uuid, default: 'uuid_generate_v1mc()' do |t|
```

Comment:
Not sure how it's been done. I expect `INSERT` needs to at least lock sequence to query the next viable id. Maybe it's cached.

Go back to our topic. Say, with a daily active users 500, a wildcard webhook could yield 1500 times. A numeric id is not dramatic different than others. The table in the webhook page shows a subsequent requests, in which UUID looks better than an numeric value as they looks different and there aren't numbered as a sequence. If there is a need for scrub, old id won't be just disappeared.

Technically, there aren't much different than an numeric incremental ID. It has a little to none chance of collision thanks to secure hash function. And `uuid_generate_v1mc` is time based so it could be used for index.

After that, speaking to Discourse instance:

UUID: good visual, don't need a sequence
Number: bad visual, leave a gap after scrub